extensions/binding/org.eclipse.smarthome.binding.onewire/src/main/java/org/eclipse/smarthome/binding/onewire/internal/OwException.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.binding.onewire.internal;
+
+/**
+ * The {@link OwException} class defines an exception for handling OneWireExceptions
+ *
+ * @author Jan N. Klug - Initial contribution
+ */
+public class OwException extends RuntimeException {
```

Comment:
Is there any particular reason why you made this a `RuntimeException` instead of a regular one? I would prefer this to be a regular exception.

Reply:
Changed. Improved Exception handling.